client/scss/components/_header.scss

```diff
@@ -57,6 +57,19 @@ header {
         .right:last-child {
             padding-right: 0;
         }
+
+        @include media-breakpoint-down(xs) {
```

Comment:
without this:

![Screenshot 2022-01-14 at 17 21 23](https://user-images.githubusercontent.com/31622/149557973-d87b7dbc-bf45-425d-a351-41c3bd1248a0.png)
![Screenshot 2022-01-14 at 17 21 32](https://user-images.githubusercontent.com/31622/149557976-9d6c15ed-5f55-4191-b117-fde2a2ff2b74.png)